spring-boot-integration/build.gradle

```diff
@@ -0,0 +1,32 @@
+buildscript {
```

Comment:
Should we write Maven configs for the examples in order to make them more accessible? Similar to what Spring does on their guides?